make-webpack-config.js

```diff
@@ -0,0 +1,172 @@
+var path = require('path');
+var fs = require('fs');
+var webpack = require('webpack');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var StatsPlugin = require('stats-webpack-plugin');
+var loadersByExtension = require('./webpack/lib/loadersByExtension');
+
+module.exports = function(options) {
+    var VERSION = require('./package.json').version;
+    var PACKAGE_NAME = require('./package.json').name;
+    var LICENSE = fs.readFileSync('LICENSE', 'utf8');
+
+    var entry = {
+        smooch: './src/js/main'
+    };
+
+    var loaders = {
+        'jsx': options.hotComponents ? ['react-hot-loader', 'babel-loader'] : 'babel-loader',
+        'js': {
+            loader: 'babel-loader',
+            include: path.join(__dirname, 'src/js')
+        },
+        'json': 'json-loader',
+        'tpl': 'ejs-loader',
+        'coffee': 'coffee-redux-loader',
+        'json5': 'json5-loader',
+        'txt': 'raw-loader',
+        'png|jpg|jpeg|gif|svg': 'url-loader?limit=10000',
+        'woff|woff2': 'url-loader?limit=100000',
+        'ttf|eot': 'file-loader',
+        'wav|mp3': 'file-loader',
+        'html': 'html-loader',
+        'md|markdown': ['html-loader', 'markdown-loader']
+    };
+    var cssLoader = options.minimize ? 'css-loader?insertAt=top' : 'css-loader?insertAt=top&localIdentName=[path][name]---[local]---[hash:base64:5]';
+    var stylesheetLoaders = {
+        'css': cssLoader,
+        'less': [cssLoader, 'less-loader']
+    };
+    var additionalLoaders = [
+        {
+            test: /\.js$/,
+            loader: 'imports?define=>false'
+        },
+        {
+            test: /src\/js\/main/,
+            loader: 'expose?Smooch'
+        }
+    ];
+    var alias = {
```

Comment:
I guess I can remove these, it was part of the boilerplate, but it's mostly the same as the alias config in requirejs.